### DISCUSSION TITLE
The shortcode "steps" doesn't work

### DISCUSSION THREAD

**Discussion by HueNa** [imfing/hextra#191] [Q&A] [answered]
Hello, 
I noticed that the ["steps" shortcode ](https://imfing.github.io/hextra/docs/guide/shortcodes/steps/) doesn't work.

To reproduce the situation, I modified the hextra-starter-template as following:
![Modified_index](https://github.com/imfing/hextra/assets/22568913/09ac80bd-ad30-4249-bfc9-8cd4fe95dc3b)
![screenshot](https://github.com/imfing/hextra/assets/22568913/463a26e3-369d-4400-8c98-7461ddfd6c64)

Can anyone show me how and where to check in order to make this work?
Any hints will be appreciated.

Thank you in advance.

I also attached the modified hextra-starter-template.
[Test.zip](https://github.com/imfing/hextra/files/13303065/Test.zip)

**Comment by imfing** [comment, marked as answer]
Can you add empty lines after the `{{% steps %}}`

```
{{% steps %}}

### Step 1

This is the first step.

{{% /steps %}}
```

**Reply by HueNa** [reply]
Wow! Comes to work VERY WELL.
Thank you very much.

**Comment by maxwelljens** [comment]
I can't seem to get it to work either, no matter how I format the code.

![image](https://github.com/imfing/hextra/assets/85795372/b2f870c7-1151-4ff8-bb15-f9e5aff39da2)

```md
Here is a short rundown of the idea; what it takes to set up your own website.

{{% steps %}}

### Step 1

Test.

### Step 2

Test.

{{% /steps %}}
```

**Reply by imfing** [reply]
Can you share a minimal example repository that reproduces your issue?

**Reply by maxwelljens** [reply]
Sure. Here you go: https://github.com/maxwelljens/steptest

I tested this exact configuration with `hugo serve`, doing nothing beyond the setup as outlined in Hextra documentation. I just populated the content pages with relevant Markdown. The codes do not work for me still.

**Reply by imfing** [reply]
@maxwelljens Please add the following settings in your `hugo.yaml`:

```yaml
markup:
  # allow raw html
  goldmark:
    renderer:
      unsafe: true
```

https://github.com/imfing/hextra-starter-template/blob/3dc5c5b971430774cc6c1c9c3b160fb6f2bb59d5/hugo.yaml#L9-L13

**Reply by maxwelljens** [reply]
Oh, thank you. That was the issue. Thanks for the quick response.
